admin_interface/templatetags/admin_interface_tags.py

```diff
@@ -65,15 +65,21 @@ def get_admin_interface_languages(context):
     return langs_data
 
 
-@simple_tag(takes_context=True)
-def get_admin_interface_theme(context):
+@simple_tag()
+def get_admin_interface_theme():
```

Comment:
The function is not using context, so I removed the parameter to make it easier to call from the next function.

Is there something I don’t understand about it?  There was a unit test related to context but the assertions were the same as for the test without so I don’t get why this tag was taking context.

Reply:
Frankly I don't remember this, probably it is not needed at all and it is caused by a copy/paste.